modular_pariah/master_files/code/game/objects/items/devices/radio/encryption_key.dm

```diff
@@ -0,0 +1,4 @@
+/obj/item/encryptionkey/heads/qm
+	name = "\proper the quartermaster's encryption key"
```

Comment:
`\proper the` 😨

(Starting the name with a lowercase letter will always automatically add a "The" in any text referencing it. `\proper the` shouldn't ever be needed unless you specifically want a lowercase "the" in every situation.)
```suggestion
	name = "quartermaster's encryption key"
```
*(also I realise this was on the original PR)*